### PR TITLE
Feature/stupid

### PR DESCRIPTION
**Pull request by AlonSchwierz**
this is a blank massage.

### CODE REVIEW

**Comment by katzuv** [comment]
> this is a blank massage.

Massage or message?

File: src/main/java/frc/robot/subsystems/Intake/Intake.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    private final IntakeIO io;
+    private final IntakeInputsAutoLogged inputs;
+
+    private Intake(IntakeIO io) {
+        inputs = new IntakeInputsAutoLogged();
+        this.io = io;
+        if (Robot.isReal()){
+
+        }
+    }
```
Comment (littlebro123) [inline src/main/java/frc/robot/subsystems/Intake/Intake.java:15]: What about getInstance? You should pass the io parameter to the constructor in the getInstance, based on whether the robot is real or not. For example:
`
public Intake getInstance() {
    if (INSTANCE == null) {
        if (Robot.isReal) {
            INSTANCE = new Intake(new IntakeIOReal());
        } else {
            // Create INSTANCE with the sim io
        }
    }
}
`

File: src/main/java/frc/robot/subsystems/Intake/IntakeIOSim.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        powerMotor.setInputVoltage(power);
+    }
+
+    public void setAngleMotor(double desiredAngleRads) {
+        angleVoltage = angleController.calculate(angleMotor.getAngleRads(), desiredAngleRads);
+        angleMotor.setInputVoltage(angleVoltage);
+    }
+
+
+}
```
Comment (littlebro123) [inline src/main/java/frc/robot/subsystems/Intake/IntakeIOSim.java:38]: Perhaps you should add some "mechanical" limits to the angle. If it reaches the top, the motor stops, and same for the bottom. You could also update the applied current to a high value when "pressed" against the "mechanical" limits (just like in the real robot).

File: src/main/java/frc/robot/subsystems/Intake/commands/Feed.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package frc.robot.subsystems.Intake.commands;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Intake.Intake;
+import frc.robot.subsystems.Intake.IntakeConstants;
+
+public class Feed extends CommandBase {
+    Intake intake;
+    double intakePower;
```
Comment (littlebro123) [inline src/main/java/frc/robot/subsystems/Intake/commands/Feed.java:8]: Please add private or public before each variable.

**Review by littlebro123 — COMMENTED** [review]
Good job.

**Pull request closed without merging**